src/Bundle/Resources/contao/dca/tl_leaflet_marker.php

```diff
@@ -78,7 +80,7 @@
                 'label'      => &$GLOBALS['TL_LANG']['tl_leaflet_marker']['delete'],
                 'href'       => 'act=delete',
                 'icon'       => 'delete.gif',
-                'attributes' => 'onclick="if(!confirm(\'' . $GLOBALS['TL_LANG']['MSC']['deleteConfirm']
+                'attributes' => 'onclick="if(!confirm(\'' . @$GLOBALS['TL_LANG']['MSC']['deleteConfirm']
```

Comment:
Please do not use `@`, use instead:

```suggestion
                'attributes' => 'onclick="if(!confirm(\'' . ($GLOBALS['TL_LANG']['MSC']['deleteConfirm'] ?? '')
```